developer/js/source/lexical-model-compiler/build-trie.ts

```diff
@@ -243,8 +233,9 @@ namespace Trie {
      * @param words a list of word and count pairs.
```

Comment:
Confirming `count` and `weight` are the same thing

Reply:
I believe they're correlated, yes.  The word count is used to determine the weight.